app/codes/blockchain.py

```diff
@@ -44,6 +44,14 @@ def get_block(self, block_index):
 
         return block
 
+    def empty_block_hash(self, block):
+        proof = 42  # Hardcoded value for empty block
```

Comment:
how is the proof incorporated inside the block? Is the block passed by reference?